modules/dynamic-plugins/rhdh-supported-plugins.adoc

```diff
@@ -15,75 +15,219 @@ Additional detail on how Red Hat provides support for bundled community dynamic
 
 There are 61 plugins available in {product}. See the following table for more information:
 
-[dynamic-plugins-matrix]
-.Dynamic plugins support matrix
+== Red Hat / Partner provided plugins
```

Comment:
@Gerry-Forde @Preeticp Is this what we're considering for the title?
```suggestion
== {company-name} or partner supported plugins
```

Reply:
I copied from https://docs.google.com/document/d/1F6ctaXcUis4ngcxGYhdPmdY23iQzQgz1v4lObn6WqZc/edit#heading=h.2mhip5hx15qi but you folks know best what copy / title you want.